pkg/proc/variables.go

```diff
@@ -516,15 +516,20 @@ func (g *G) Go() Location {
 }
 
 // StartLoc returns the starting location of the goroutine.
-func (g *G) StartLoc() Location {
-	f, l, fn := g.variable.bi.PCToLine(g.StartPC)
-	return Location{PC: g.StartPC, File: f, Line: l, Fn: fn}
+func (g *G) StartLoc(tgt *Target) Location {
+	fn := g.variable.bi.PCToFunc(g.StartPC)
+	fn = tgt.dwrapUnwrap(fn)
```

Comment:
Do we have test coverage for a goroutine created within a deferred function (which ends up wrapped on >= 1.17)?

Reply:
TestDwrapStartLocation added by this.